vms/avm/client.go

```diff
@@ -385,6 +389,12 @@ func (c *client) GetAllBalances(
 	return res.Balances, err
 }
 
+func (c *client) GetTxFee(ctx context.Context, options ...rpc.Option) (*GetTxFeeReply, error) {
```

Comment:
Would it be cleaner to return `(uint64, uint64, error)`? Since there aren't that many values, I feel like the usage would actually be nicer without returning the struct.